corehq/apps/linked_domain/templates/linked_domain/tabs/push_release_content_tab.html

```diff
@@ -2,6 +2,14 @@
 {% load i18n %}
 
 <div id="ko-tabs-push-content" class="ko-template">
+  <h3>{% trans "Push Content" %}</h3>
+  <p>
+    {% blocktrans %}
+      Prepare a new release by selecting which data models you would like to push to the selected downstream project spaces.<br/>
+      <a href="#" target="_blank">Learn more</a> about pushing release content with linked projects.
```

Comment:
confirming that these docs aren't ready yet?

Reply:
Yeah pretty sure public facing docs are not ready. I'll make a ticket to make sure I address these.